### ISSUE TITLE
Small problem: calculating death rates

### ISSUE THREAD

**Issue by PenelopeFudd** [CSSEGISandData/COVID-19#23]
It's tempting to try and calculate how deadly a virus is by just dividing the number of people who have died by the number of people who have died plus the number who have recovered, but there's a problem with the number you get: it's too high.

The calculation needs to compare people who were infected at the same time, because it takes longer to recover than it does to die.

Eventually the simple calculation and the complicated calculation will converge, but we're not there yet.  :-/

**Comment by naterattner** [comment]
Is there a reason that `number died / total confirmed cases` would not work?

**Comment by smiletwchien** [comment]
The association between the died(on Axis Y) and the recovered(on Axis X) might be enough to satisfy the need we concern at  http://www.smilechien.idv.tw/kpiall/deathrecovery.asp

**Comment by PenelopeFudd** [comment]
What a person who has just become infected wants to know is: What are my chances of dying?

The easiest way is to wait until everyone has either recovered or died, then `number died / total confirmed cases` would be exactly correct.  The problem is, we can't wait that long.  

The thing to do instead is to examine a group of patients (a 'cohort') who all got sick at the same time, and who are no longer sick (one way or the other).  In that group, we can do the calculation `number died / total confirmed cases` and get the right result.

That said, this *really* only shows the chances of dying if you were infected at the same time as the cohort.  We'll still want to rerun the calculations with every cohort we can get, and see if there are any patterns, such as 'certain treatments work better/worse', or 'the virus is getting deadlier/safer over time', or 'people in a certain age range are more/less vulnerable', etc.

**Comment by PenelopeFudd** [comment]
Here's an article comparing this virus to Ebola and Influenza, from National Geographic: https://www.nationalgeographic.com/science/2020/02/graphic-coronavirus-compares-flu-ebola-other-major-outbreaks/

From that, it looks like Influenza has killed many more people than Ebola, MERS, SARS and COVID-19 (this virus) combined.

**Comment by jmcastagnetto** [comment]
You might want to take a look at https://www.worldometers.info/coronavirus/coronavirus-death-rate/

**Comment by googled** [comment]
All deaths are accounted for but not all infected as some people at home might have been asymptomatic or had recovered by themselves. The number of infected people is believed to be as high as 5 fold the numbers we currently have, therefore making the death-to-case ratio under 1%.

**Comment by Fromalaska49** [comment]
While enlightening, I believe the analysis and discussions here are beyond the scope of data sharing, the purpose of this repository.

**Comment by avatorl** [comment]
http://avatorl.org/en/novel-coronavirus-2019-ncov-fatality-rate-who-and-media-vs-reality/

**Comment by JHale716** [comment]
The lag factor is problematic, an approach that could be considered is to compare on a infections X days ago against the deaths we know of today. Unlikely to be accurate, however, a better ‘guide’ to the trend than the raw today approach which hides the irate with the exponential new cases growth. 

From the media; reports on progress this spread being 3-7 days gives an interesting picture, as too there is further lag with testing for confirmation creating up to a 48 hour lag on confirmation from presentation to confirmation. Then we have the new CT approach which now shortcuts that delay, making the data inconsistent if you are pursuing absolutes.

This is assuming data accuracy with complete reporting and no added manipulation. I have more confidence with the international data than I do with the Chinese data. Especially when we have reports that the death rate may not be being reported accurately as well as what has been said in this thread about infection data. We do what we do with what we have.

**Comment by PenelopeFudd** [comment]
Yes: as time goes on, the lag will have less and less effect on the overall rate.

**Comment by designerfuzzi** [comment]
when the virus is not infecting more people the death rate data would drop again of course.
So any calculation is bound to the situation when the virus is still going around.

Because cases who end up dead earlier than cases recovering there should obviously a time where the time from being confirmed infected to death can be compared to the cases where the same time passed but the case survived/recovered. Both are treated usually in clinical circumstances, otherwise they where not confirmed cases. So any calculation based on a comparison of both inside this particular time span are expression of health care quality and lots of other factors. But you can compare them. It would express death rate even when confirmed inside a health care system.
 It doesnt make sense to declare that cases without symptoms (unconfirmed cohort) are inside the cohort just to drop the mortality rate. While cases inside confirmed cohort are usually under  health care treatment. You could include the influenza cohort and watch its axis curve, if its double the normal height then you know that one or the other has a higher rate. This is because the covid-19 cohort is new, not even a year old while influenza data is well known.
